addons/api/addon/services/indexed-db.js

```diff
@@ -0,0 +1,40 @@
+import Service from '@ember/service';
+import Dexie from 'dexie';
+import { inject as service } from '@ember/service';
+
+// List of resources we support to be cached and searched in
+// indexedDb. Any field that can be searched upon should be added to
+// this index. Increment the version number of the database whenever
+// indexes change.
+export const modelIndexes = {
+  target: '&id, attributes.type',
```

Comment:
what does `&id` mean?

Reply:
Ah good question, this just indicates as part of the index that the field is unique, take a look at the docs [here](https://dexie.org/docs/Version/Version.stores()#schema-syntax) for some examples